src/Mobs/Path.cpp

```diff
@@ -57,7 +57,7 @@ cPath::cPath(
 	m_Destination.y = FloorC(a_EndingPoint.y);
 	m_Destination.z = FloorC(a_EndingPoint.z - HalfWidthInt);
 
-	if (GetCell(m_Source)->m_IsSolid || GetCell(m_Destination)->m_IsSolid)
+	if (!IsWalkable(m_Source))
```

Comment:
Explanation: The mob should pathfind even if target is not reachable because the pathfinder can pick a nearby spot.